test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -40,18 +40,18 @@ public void TestPfiRegressionOnDenseFeatures()
             // X4Rand: 3
 
             // For the following metrics lower is better, so maximum delta means more important feature, and vice versa
-            Assert.True(MinDeltaIndex(pfi, m => m.L1) == 3);
-            Assert.True(MaxDeltaIndex(pfi, m => m.L1) == 1);
+            Assert.True(MinDeltaIndex(pfi, m => m.L1.Mean) == 3);
```

Comment:
>Mean [](start = 53, length = 4)

Please validate all the metrics statistical measures that you expose. You are only checking mean for now. Do it at least in one case. (no need to do it everywhere.)

Reply:
Done.